osu.Game/Users/UserStatistics.cs

```diff
@@ -26,17 +26,22 @@ public struct LevelInfo
             public int Progress;
         }
 
-        [JsonProperty(@"pp")]
-        public decimal? PP;
+        [JsonProperty(@"rank")]
+        public UserRanks Ranks;
 
-        [JsonProperty(@"pp_rank")] // the API sometimes only returns this value in condensed user responses
-        private int? rank
+        // eventually UserRanks object will be completely replaced with separate global and country rank properties, see https://github.com/ppy/osu-web/blob/cb79bb72186c8f1a25f6a6f5ef315123decb4231/app/Transformers/UserStatisticsTransformer.php#L53.
+        // but for now, always point UserRanks.Global to the global_rank property, as that is included solely for requests like GetUsersRequest.
+        [JsonProperty(@"global_rank")]
+        private int? globalRank
         {
             set => Ranks.Global = value;
         }
 
-        [JsonProperty(@"rank")]
-        public UserRanks Ranks;
+        [JsonProperty(@"pp")]
+        public decimal? PP;
+
+        [JsonProperty(@"pp_rank")]
+        public int PPRank;
```

Comment:
I've intentionally made this an own property instead of redirecting its value to `Ranks.Global`, as the newly added `global_rank` property serves the same purpose, and it looks to be included in every user statistics API responses as well (`GetUserRequest`, `GetUsersRequest`, `GetUserRankingsRequest`).

I've ensured there were no components affected by this change, but all `Ranks.Global` usages were coming from responses of `GetUserRequest` (where `rank` would be included), i.e. there were no components relying on the `pp_rank` property, so nothing should break there.

Reply:
This isn't being used anywhere, is there a reason to keep it around? If we are, it should probably not be called `PPRank` since that's quite nondescript.

Reply:
There are no usages to it and can be removed, just felt like it may be used further on for getting an "absolute global rank based on pp".

Also yeah not sure about the name either, just took it directly from the property itself, maybe `RankInPP` or `AbsoluteRank`.

Reply:
What does "absolute global rank based on pp" mean? As opposed to what? When is rank not based on pp?

Reply:
Ignore `in pp`, it's just that this PP rank *presumably* has the user's absolute global rank, regardless of whether the user plays on the ruleset or not, opposed to `global_rank` which returns `null` if the user *isn't ranked* in said ruleset.

Reply:
I don't get it.

Reply:
This looks like a left-over on osu-web's end. We'll remove `pp_rank` in the next web release, so it can be removed here as well.